vue/components/forms/personalization-form/initials-images.vue

```diff
@@ -215,6 +215,14 @@ export const InitialsImages = {
             if (this.imageBorderRadius) base["border-radius"] = `${this.imageBorderRadius}`;
             if (this.imageObjectFit) base["object-fit"] = this.imageObjectFit;
             return base;
+        },
+        _openExternally() {

```

Comment:
we normally don't use the private prefix `_` in computed, so I think you can rename it to `open` or something else